src/datascience-ui/variable-view/variableViewPanel.tsx

```diff
@@ -118,7 +118,7 @@ export class VariableViewPanel extends React.Component<IVariableViewPanelProps>
 
     private pageInVariableData = (startIndex: number, pageSize: number) => {
         this.props.getVariableData(
-            this.props.currentExecutionCount,
+            this.props.variableState.currentExecutionCount,
```

Comment:
This code I had actually copied over from the interactivePanel code. Was causing all the page in issues. With my code now I'm only updating and using the variableState from the main state. This actually looks to me like it might be "wrong" in the existing old editor code, but I didn't want to change that for no reason.

Reply:
Think that it worked fine in the old code as both main state currentExecutionCount and variableState.currentExecutionCount were in sync. But since I'm just using variable state main state currentExecutionCount was not getting updated.

Reply:
So all pageIn data was initially getting fetched with 0.